examples/internet_identity/src/frontend/index.ts

```diff
@@ -1,6 +1,6 @@
 import { Identity } from '@dfinity/agent';
 import { AuthClient } from '@dfinity/auth-client';
-import { toJwt } from 'azle/client';
+import { toJwt } from 'azle/http_client';
```

Comment:
Also why didn't the type check test catch this in previous prs? And if it did why didn't it get fixed there? And if it did, which I thought it did, why is it getting fixed again? I'm very confused about this

Reply:
I think we have talked about this a lot, so I think this can be resolved. If I remember correctly tests were breaking and that's why I came here to fix it...or something like that